billingClient/billingClient.go

```diff
@@ -0,0 +1,42 @@
+package billingclient
+
+import (
+	"context"
+
+	e3dbClients "github.com/tozny/e3db-clients-go"
+	"github.com/tozny/e3db-clients-go/authClient"
+)
+
+const (
+	BillingServiceBasePath = "v1/billing"
+	InternalCreateCustomer = "/internal/create_customer"
```

Comment:
Unless you forsee this changing a lot/being used by consumers of this package I don't thank this needs to be a constant or public.